src/components/PersonList/PersonList.tsx

```diff
@@ -0,0 +1,186 @@
+import TabContext from '@mui/lab/TabContext'
+import TabList from '@mui/lab/TabList'
+import TabPanel from '@mui/lab/TabPanel'
+import { Skeleton } from '@mui/material'
+import Tab from '@mui/material/Tab'
+import { Box } from '@mui/system'
+import { User } from 'fluro'
+import { ReactElement, useMemo, useState } from 'react'
+import { useQuery } from 'react-query'
+
+import { getContacts } from '../../lib/queries/getContacts'
+import { GetContacts } from '../../lib/queries/getContacts/getContacts'
+import { getProcessDefinitions } from '../../lib/queries/getProcessDefinitions'
+import { PersonListItem } from '../PersonListItem'
+
+interface Processes {
+  // Where string[] is the array of contactIds
+  [processType: string]: string[]
+}
+
+interface PersonListProps {
+  search: string
+  user: User
+}
+
+function getAllProcesses(contacts: GetContacts[] | undefined): Processes {
+  const allProcesses: Processes = { all: [] }
+
+  if (contacts != null) {
+    contacts.forEach((contact) => {
+      const processTypes = Object.keys(contact.process)
+      const processes = Object.values(contact.process)
+
+      processTypes.forEach((type) => {
+        if (allProcesses[type] == null) {
+          allProcesses[type] = []
+        }
+      })
+      processes.forEach((process) => {
+        allProcesses.all.push(process.item)
+        allProcesses[process.definition].push(process.item)
+      })
+    })
+  }
+
+  return allProcesses
+}
+
+export function PersonList({ search, user }: PersonListProps): ReactElement {
+  const [processType, setProcessType] = useState('all')
+
+  const { data: definitions, isLoading: isDefinitionLoading } = useQuery(
+    ['definitions', { type: 'process' }],
+    getProcessDefinitions
+  )
+
+  const { data: contacts, isLoading: isContactLoading } = useQuery(
+    ['contacts', { user: user.contacts[0] }],
+    getContacts(user.contacts[0])
+  )
+
+  const allProcesses = useMemo(() => {
+    return getAllProcesses(contacts)
+  }, [contacts])
+
+  const processTypes = Object.keys(allProcesses)
+  const oneProcessType = processTypes.length < 3
+
+  const handleChange = (e: React.SyntheticEvent, newValue: string): void => {
+    setProcessType(newValue)
+  }
+
+  return (
+    <TabContext value={processType}>
+      <Box sx={{ borderBottom: 1, borderColor: 'divider' }}>
```

Comment:
A divider immediately after? I was following material documentation 
<img width="596" alt="image" src="https://user-images.githubusercontent.com/10802634/180124459-d4126ba6-7408-49b6-b9c1-f054b1610f71.png">
but can update

Reply:
<img width="414" alt="image" src="https://user-images.githubusercontent.com/10802634/180324879-9f5ef5a7-f2fd-4751-bf94-fb84b679224a.png">
Ah, the reason they use box is because the Divider leaves a small gap - I'm not keen to use negative margins to fix it, so going to leave it like the example